ledger/api-server-damlonx/reference-v2/BUILD.bazel

```diff
@@ -166,7 +166,6 @@ server_conformance_test(
     test_tool_args = [
         "--all-tests",
         "--exclude=LotsOfPartiesIT",
-        "--exclude=TimeIT",
         "--exclude=ConfigManagementServiceIT",
     ],
```

Comment:
Perhaps just drop the arguments here and run the defaults. The `TransactionScale` test doesn't really prove anything in this project.

Reply:
@SamirTalwar why do you say the `TransactionScale` test doesn't really prove anything?

Reply:
Because this is a reference implementation which is going to be removed soon. Which is also an argument for leaving it as it is… so carry on either way. 🙂

Reply:
Addressed by https://github.com/digital-asset/daml/commit/29be7bbbd0c2e58ada6be5484c1bf21d71438528